src/thunderbots/software/backend_output/grsim/grsim_backend.cpp

```diff
@@ -1,6 +1,5 @@
 #include "backend_output/grsim/grsim_backend.h"
 
-#include <chrono>
 #include <ctime>
```

Comment:
Sorry if my last comment was unclear. We want to keep `<chrono>`, and remove `<ctime>`

Reply:
Chrono kept ctime removed.